src/test/java/org/folio/rest/impl/AccountsAPITest.java

```diff
@@ -136,8 +143,8 @@ public void testAllMethodsAndEventPublishing() {
 
     Account accountToDelete = new Account()
       .withId(accountId)
-      .withRemaining(new MonetaryValue(new BigDecimal("0.00")));
-
+      .withRemaining(new MonetaryValue(new BigDecimal("0.00")))
+      .withDateClosed(new Date());
```

Comment:
We are not checking date closed in assertion later. Please find a way to do that.